status/versions.go

```diff
@@ -312,18 +320,52 @@ func kubernetesVersion() (*ExternalServiceInfo, error) {
 // did that that version change will matter, and the kiali pod could be bounced as a workaround.
 var istioSupportsCanonical *bool
 
-// IstioSupportsCanonical returns true if Istio version can be determined and is >= 1.5.
-// TODO: This test can be removed when Kiali stops supporting any Istio versions < 1.5
+// IstioSupportsCanonical returns true if Telemetry V2 is enabled
+// TODO: This test can be removed when Kiali stops supporting Istio versions with Mixer Telemetry
 func IstioSupportsCanonical() bool {
```

Comment:
Perhaps this function can be renamed in something like `IsMixerEnabled`, to be more relevant for other uses.

Reply:
I like the idea of a IstioMixerEnabled() but it probably should be in addition to IstioCanonicalSupported() because it's possible in versions < 1.5 to run telem V2 (experimental) but without canonical fields, which were introduced in 1.5.  In short the test for canonical is (!mixer && version >= 1.5).